thirdparty/patches/grpc-python.patch

```diff
@@ -1,4 +1,14 @@
+From 30bd860bd8a8bb223f3aab86afc2ed28c5bd5a7d Mon Sep 17 00:00:00 2001
+From: acxz <17132214+acxz@users.noreply.github.com>
+Date: Thu, 17 Feb 2022 11:07:42 -0500
+Subject: [PATCH] fix python command on windows
+
+---
+ third_party/py/python_configure.bzl | 2 +-
```

Comment:
Can you tell me what this patch change is for?

Reply:
Oh my bad, looks like somebody already fixed the issue. Sorry about that, left over from previous commit.